patches/0016-user-agent-set-to-Pulumi-apn-1.1-info.patch

```diff
@@ -0,0 +1,73 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: corymhall <43035978+corymhall@users.noreply.github.com>
+Date: Thu, 11 Apr 2024 08:06:27 -0400
+Subject: [PATCH] user agent set to Pulumi apn 1.1 info
```

Comment:
When you create the commit you can add a commit message with details as well. Can you add additional details to the commit so that in the future we can see all the context in the patch?

e.g.

- Why do we need to do this?
- Where did you get the APN id?
- ...